conf/modules/mag_rm3100.xml

```diff
@@ -44,6 +45,7 @@
     <configure name="MAG_RM3100_I2C_DEV" case="upper|lower"/>
     <define name="USE_$(MAG_RM3100_I2C_DEV_UPPER)"/>
     <define name="MAG_RM3100_I2C_DEV" value="$(MAG_RM3100_I2C_DEV_LOWER)"/>
+    <define name="MODULE_RM3100_RATE" value="60"/>
```

Comment:
Why this define here if you do nothing with it?

Reply:
I added it there too because otherwise if the user does not specify the variable in the airframe XML the compilation throws an error. This way it at least defaults to the old value of 60 Hz